### PR TITLE
Fix SENDING_MESSAGE_START payload mismatch

### PR DESCRIPTION
**Pull request by sravan-s**
* SENDING_MESSAGE_START payload didnt had channel object before
* Fix CSS issue with render custom message

### CODE REVIEW

File: src/smart-components/OpenchannelConversation/index.tsx [hunk 1]
```diff
@@ -306,11 +306,15 @@ export const OpenchannelConversation = (props: Props): JSX.Element => {
       {
         renderMessageInput
           ? (
-            renderMessageInput({
-              channel: currentOpenChannel,
-              user: user,
-              disabled: disabled,
-            })
+            <div className="sendbird-openchannel-footer">
```
Comment (HoonBaek) [inline src/smart-components/OpenchannelConversation/index.tsx:309]: Is there no design issue after this change?

File: src/smart-components/OpenchannelConversation/index.tsx [hunk 2]
```diff
@@ -202,7 +202,7 @@ export const OpenchannelConversation = (props: Props): JSX.Element => {
       if (channel && (channelUrl === channel.url)) {
         messagesDispatcher({
           type: messageActionTypes.SENDING_MESSAGE_START,
-          payload: message,
+          payload: { message, channel },
```
Comment (HoonBaek) [inline src/smart-components/OpenchannelConversation/index.tsx:205]: Need to add de-structuring assignment in reducer to fit this object

File: src/smart-components/OpenchannelConversation/index.tsx [hunk 3]
```diff
@@ -212,7 +212,7 @@ export const OpenchannelConversation = (props: Props): JSX.Element => {
       if (channel && (channelUrl === channel.url)) {
         messagesDispatcher({
           type: messageActionTypes.SENDING_MESSAGE_SUCCEEDED,
-          payload: message,
+          payload: { message, channel },
```
Comment (HoonBaek) [inline src/smart-components/OpenchannelConversation/index.tsx:215]: same here

File: src/smart-components/OpenchannelConversation/index.tsx (hunk 1 above)
Comment (sravan-s) [reply src/smart-components/OpenchannelConversation/index.tsx:309]: There was issue before 😆

Comment (HoonBaek) [reply src/smart-components/OpenchannelConversation/index.tsx:309]: Oh okok haha

**Review by HoonBaek — APPROVED** [review]
LGTM

**Pull request merged**